### PR TITLE
Update AboutUs.md and  project title in _base.scss

### LINKED ISSUE #11: Update site-wide settings and update AboutUs page
- Update site-wide settings
- Update Yu Zher's AboutUs page

### PR DESCRIPTION
**Pull request by LZher385**
Changes:

- Update AboutUs.md and add profile picture
- Update project title in _base.scss

Issues:
Fixes #11

### CODE REVIEW

File: docs/AboutUs.md [hunk 1]
```diff
@@ -43,17 +43,17 @@ You can reach us at the email `seer[at]comp.nus.edu.sg`
 <img src="images/johndoe.png" width="200px">
 
 [[github](http://github.com/johndoe)]
-[[portfolio](team/johndoe.md)]
+[[portfolio](team/lzher385.md)]
```
Comment (YeluriKetan) [inline docs/AboutUs.md:46]: It appears that this file is causing merge conflicts that are beyond limits to resolving on Github directly. Perhaps you could update this file with the current latest version from the master branch and commit, after which I can merge them.

File: docs/team/lzher385.md [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+gitGud is a desktop application for managing friends' gaming information. gitGud uses a simple Graphical User Interface (GUI) combined with an optimized Command Line Interface (CLI) to give you a smooth and efficient experience. If you can type fast enough, gitGud can get you contact management tasks done faster than traditional GUI-based applications.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to link a friend with a game he/she plays.
+  * What it does: allows the user to associate a friend in his contact list with a game that was previously recorded.
+  * Justification: This feature improves the product significantly because a user is able to note down what games his/her friends play.
+  
+
+* **New Feature**: Added a history command that allows the user to navigate to previous commands using up/down keys.
```
Comment (YeluriKetan) [inline docs/team/lzher385.md:17]: I don't think this is a valid feature we are planning to implement as of now. Can perhaps leave this out for later?

**Review by YeluriKetan — CHANGES_REQUESTED** [review]
Mostly good to merge. Minor fixes for conflict issues.

File: docs/team/lzher385.md (hunk 2 above)
Comment (LZher385) [reply docs/team/lzher385.md]: My bad it was a line from the original template that I forgot to delete. Thanks for pointing it out!

**Review by YeluriKetan — APPROVED** [review]
LGTM. No issues

**Pull request merged**